.github/workflows/lintrest.yaml

```diff
@@ -18,6 +18,6 @@ jobs:
       - uses: actions/setup-go@v2
         with:
           go-version: '1.14.5'
-      - run: bash scripts/gogetcookie.sh
-      - run: make tools
-      - run: GOGC=5 make lintrest
+      - uses: golangci/golangci-lint-action@v2
```

Comment:
does this assume `.golangci.ym` ? we can probably rename this step to golint rather then lint rest if it actually manages to run the full linting suite